### PR TITLE
WIP [Blockchain] Ethereum I

### PR DESCRIPTION
**Pull request by EdgarBarrantes**
Issue: #562
Done: 
- [x] ethereum-and-ether
- [ ] what-does-ethereum-do-differently
- [ ] what-are-smart-contracts

Notes: 
Needs more insights.
This is a work in progress.

### CODE REVIEW

**Comment by enki-hq** [comment]
Failed parsing files: 
  - blockchain/fundamentals/ethereum-i/ethereum-and-ether.md: 
 > Must have exactly 1 headline but found 2 instead.

File: blockchain/fundamentals/ethereum-i/ethereum-and-ether.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Ethereum and Ether
```
[the remark this answers is not in the document]
Comment (mihaiberq) [reply blockchain/fundamentals/ethereum-i/ethereum-and-ether.md]: Sorry, that was my fault. I am updating the branch with the new format today. @EdgarBarrantes you should fetch the changes before you continue working on this one.

**Comment by enki-hq** [comment]
Failed parsing files: 
  - blockchain/fundamentals/ethereum-i/ethereum-and-ether.md: 
 > bad indentation of a sequence entry at line 23, column 15:
      - [Ethereum](https://www.ethereum.org/){website}
                  ^

**Comment by enki-hq** [comment]
Rule insight:author failed for files: 
  - blockchain/fundamentals/ethereum-i/ethereum-and-ether.md

**Comment by enki-hq** [comment]
Rule workout:insights failed for files: 
  - blockchain/fundamentals/ethereum-i/README.md

**Comment by enki-hq** [comment]
Rule workout:insights failed for files: 
  - blockchain/fundamentals/ethereum-i/README.md

File: blockchain/fundamentals/ethereum-i/ethereum-and-ether.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (EdgarBarrantes) [reply blockchain/fundamentals/ethereum-i/ethereum-and-ether.md]: Thanks @Loopiezlol! And yeah, I just did @mihaiberq, thanks! 
I'm still getting this error "Workout "insights" must exist in database or as new files."
I believe that's an error in the README.md of the workout but i'm not sure what it is. Could you point me in the right direction?

**Comment by ryanwarsaw** [comment]
Stale and conflicts with master, going to close. Thanks for the PR though :)

**Pull request closed without merging**